src/components/Hero/hero.scss

```diff
@@ -0,0 +1,131 @@
+@import '../../styles/variables';
+
+.hero__container{
```

Comment:
rename classes and use classical scss structure:
.hero{
  &__title{
    ...styles for hero__title
  }
 &__tumb{
...styles for hero__thumb
  }
}

Reply:
done